### ISSUE TITLE
Suggestion: Add a simple configuration file

### ISSUE THREAD

**Issue by crepassob** [Vonr/parui#5]
I have been using parui for a while and it works pretty well, but I think it would be nice to add a simple configuration file to modify some of the default settings.
For example, I often use paru -Rs "package name" when removing packages, but since the parameters are written into the code, I have to modify the [code](https://github.com/Vonr/parui/blob/4d84849cce0f3a53a887236e297517552cd2b774/src/main.rs#L652) to make it work as intended.
For another example, sometimes after I install a package, I will continue to install other packages, but after each installation, parui will exit, which makes me have to open it again, which is a bit annoying.
So I think it is necessary to add a simple configuration file to modify these default behaviors, and considering that this project may add more features in the future, writing some parameters and options in the code is not a good option.

**Comment by Vonr** [comment]
feel free to make a PR - but for your second issue, you can select multiple packages with Space and install/remove them all at once.

**Comment by crepassob** [comment]
Thanks for your response, I can make a PR about the first question, but this is not my core point. My core point is to suggest adding a configuration file to modify the default behavior in a more user-friendly way. Just as paru can use paru.conf to modify settings.

**Comment by Vonr** [comment]
yup, i understood, but unfortunately I don't have much time to work on parui and the defaults work for me right now. will still accept any PRs for implementing the config file though.

**Comment by ghost** [comment]
that would be really good because i want to change colors

**Comment by UffeJakobsen** [comment]
👍  for config file
